CMakeLists.txt

```diff
@@ -196,8 +196,8 @@ set(_acts_autodiff_version 0.6)
 set(_acts_boost_version 1.71.0)
 set(_acts_dd4hep_version 1.21)
 set(_acts_edm4hep_version 0.7)
-set(_acts_geomodel_version 4.6.0)
 set(_acts_eigen3_version 3.3.7)
+set(_acts_geomodel_version 5.6.0)
```

Comment:
It's becoming soon 6.0.0

Reply:
Will update at some point in the future when 6.0.0 is available